Planets and Systems/Harr'qac.md

```diff
@@ -36,52 +36,103 @@
+
 ### Living
 
-Harr'qaci Skrell often build circular houses, which often have many entrances around the ground floor. Usual living house for one family has 4 floors, the ground floor being entrance (since doors are at least on four sides), which can be also used for less private meetings, celebrations and stuff. First floor usually houses kitchen, living room and toilet. Second floor contains private rooms or offices and usually a toilet too. Third floor is mostly the same as second, but less usually has a toilet. In he middle of each floor, there is either an elevator or a staircase. On the top of houses, there are balconies, in which in some cases Skrell have little gardens. There's also a new trend currently, to build narrow metal bridges between balconies of houses with simillar height, if the inhabitants like each other.
+Harr'qaci Skrell usually build circular houses, which often have many entrances around the ground floor. The usual living house for one family has 4 floors, the ground floor being entrance (since doors are at least on four sides), which can be also used for less private meetings or celebrations. First floor usually houses the kitchen, living room and toilet. Second floor contains private rooms or offices and usually a toilet too. Third floor is mostly the same as second, but it doesn't usually have a toilet. In he middle of each floor, there is either an elevator or a circular staircase. On the top of the houses, there are balconies, in which some Skrellian families have little gardens. There's also a new trend currently spreading in Skrriol and Qrri'Mal, to build narrow metal bridges between balconies of houses with simillar height, if the families are like meeting each other.
 
 ## Government
 
-Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac.
+Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar, yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac and are very respected amongst the population.
```

Comment:
Sure, I'm going to invent something new. Perhaps I'll use this prefix, not sure yet.